### PR TITLE
Adding best practice for policy generator 2.4

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25740#issuecomment-1279102380

### CODE REVIEW

**Review by oafischer — APPROVED** [review]
Would it be better to have the tag in a separate line?

**Comment by mprahl** [comment]
@dockerymick that's not quite right because where you made the change is on the version of the example operator that we are generating a policy for.

I would probably just add a sentence under the `2.4.2. Policy generator` section that states the bundled/included Policy Generator version.

**Comment by dockerymick** [comment]
@mprahl thanks for your review, i just made an update. When you have an opportunity, will you rereview?

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 The policy generator is a part of the {product-title} application lifecycle subscription GitOps workflow that generates {product-title} policies using Kustomize. The policy generator builds {product-title} policies from Kubernetes manifest YAML files, which are provided through a `PolicyGenerator` manifest YAML file that is used to configure it. The policy generator is implemented as a Kustomize generator plugin. For more information on Kustomize, see the link:https://kustomize.io/[Kustomize documentation]. 
 
+The latest version of the policy generator is included in the installation. The policy generator version used is v1.4.1.
```
Comment (mprahl) [inline governance/policy_generator.adoc]:
```suggestion
The Policy Generator version bundled in this version of RHACM is v1.4.1. 
```

**Comment by dockerymick** [comment]
@mprahl thanks for your suggestion, made an update. If you are ok with that update, i will merge and apply similar changes to the other versions

File: governance/policy_generator.adoc [hunk 2]
```diff
@@ -142,9 +144,10 @@ spec:
   name: openshift-gitops-operator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
+  startingCSV: openshift-gitops-operator.v<version>
```
Comment (mprahl) [inline governance/policy_generator.adoc]: I would like the example to be valid without user alteration.
```suggestion
```

**Review by mprahl — APPROVED** [review]

File: governance/policy_generator.adoc [hunk 3]
```diff
@@ -144,7 +146,7 @@ spec:
   sourceNamespace: openshift-marketplace
 ----
 
-To pin to a specific version of the operator, you can set the `spec.startingCSV` value to `openshift-gitops-operator.v1.2.1` (replace `v1.2.1` with your preferred version).
+To pin to a specific version of the operator, you can set the `spec.startingCSV` value to `openshift-gitops-operator.v<version>`. Replace `<version>` with your preferred version.
```
Comment (dockerymick) [inline governance/policy_generator.adoc]: @mprahl since you removed the parameter, is this statement accurate or should it be removed?

Comment (mprahl) [reply governance/policy_generator.adoc]: Yes, I think it still applies. Perhaps you can change the word `set` to `add` instead?

**Comment by dockerymick** [comment]
@oafischer @mprahl thanks so much for your reviews. I applied the suggestions and ready to merge. Also making similar updates to the other versions

**Pull request merged**